src/Serilog.Sinks.ColoredConsole/project.json

```diff
@@ -1,5 +1,5 @@
 ﻿{
-  "version": "2.0.0-rc-*",
+  "version": "2.0.0-*",
```

Comment:
Under the current build infrastructure this needs to be `2.0.0` unfortunately (otherwise pre-release packages are built). We're aiming to update builds shortly, now that the runtime is stable.